app/pages/sections/catalogue-solutions/routes.test.js

```diff
@@ -36,6 +37,8 @@ const mockSelectedSolutionIdCookie = 'selectedSolutionId=solution-1';
 const mockSelectedRecipientIdCookie = 'selectedRecipientId=recipient-1';
 const mockSelectedPriceIdCookie = 'selectedPriceId=1';
 
+const mockGetPageDataCookie = `orderItemPageData=${JSON.stringify({})}`;
```

Comment:
Is the JSON.stringify needed here? Could the value just be set to '{}'?

Reply:
good point. I don't think it is needed